rust/core-lib/src/plugins/manager.rs

```diff
@@ -99,7 +99,8 @@ impl PluginManager {
                 let buffers = self.buffers.clone().to_weak();
                 let mut plugin_ref = plugin.clone();
 
-                plugin.update(&update, move |response| {
+                plugin.update(&update);
+                /*plugin.update(&update, move |response| {
```

Comment:
@cmyr What was being done in this method earlier? Are similar changes required somewhere else now.

Reply:
No, this is not currently used and will be removed shortly.